tilavarauspalvelu/management/commands/data_creation/create_caisa.py

```diff
@@ -49,734 +54,686 @@ def _create_caisa(metadata_sets: dict[SetName, ReservationMetadataSet]) -> None:
         phone="+358 9 310 37500",
     )
 
+    ###########################################################################################################
+    # Get metadata sets
+    ###########################################################################################################
+
+    metadata_sets: dict[SetName, ReservationMetadataSet] = {
+        SetName(meta.name): meta for meta in ReservationMetadataSet.objects.all()
+    }
+
     ###########################################################################################################
     # Create tax percentages and payment types
     ###########################################################################################################
 
-    tax_percentage_0, _ = TaxPercentage.objects.get_or_create(value=0.0)
-    tax_percentage_24, _ = TaxPercentage.objects.get_or_create(value=24.0)
-    payment_type_online, _ = ReservationUnitPaymentType.objects.get_or_create(code="ONLINE")
+    tax_percentage_0 = TaxPercentageFactory.create(value=0.0)
+    tax_percentage_24 = TaxPercentageFactory.create(value=24.0)
```

Comment:
The tax percentage could be 25.5 now. (Matches test environment data, then.)

Reply:
Fixed.